src/test/java/org/opentripplanner/routing/algorithm/mapping/BikeRentalSnapshotTest.java

```diff
@@ -27,9 +34,15 @@
 
     @BeforeAll
     public static void beforeClass() {
+        Locale.setDefault(Locale.US);
         loadGraphBeforeClass();
     }
 
+    @AfterAll
+    public static void afterClass() {
+        Locale.setDefault(DEFAULT_LOCALE);
```

Comment:
I don't think this is necessary anymore, isnt' it?

Reply:
It might not be, but I think I would prefer to make any further changes to the locale handling in its own PR.